### DISCUSSION TITLE
Make Phoenix a PWA

### DISCUSSION THREAD

**Discussion by RoKrypto** [phcode-dev/phoenix#575] [Ideas]
It would be cool to have it available as a PWA so as to take advantage of the features both possess, its nature as an open source IDE and its complements features as a PWA.

**Comment by abose** [comment]
@RoKrypto Thanks for the feature suggestion. Will work on PWA as soon as JS code intelligence and extension store is done in phoenix.

**Comment by davidlimonche** [comment]
> Sería genial tenerlo disponible como PWA para aprovechar las características que ambos poseen, su naturaleza como IDE de código abierto y sus características complementarias como PWA.

estoy de acuerdo con la idea
